src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 if values < 15`, () => {
```

Comment:
The test description is not clear. Test descriptions should be written in a way that they clearly describe what functionality is being tested.

Reply:
Done